netbox/utilities/filters.py

```diff
@@ -143,6 +144,16 @@ def filter(self, qs, value):
         return qs.distinct() if self.distinct else qs
 
 
+class NullableMultipleChoiceFilter(django_filters.MultipleChoiceFilter):
+    """
+    Similar to NullableCharFieldFilter, but allows multiple values including the special NULL string.
+    """
+    def filter(self, qs, value):
+        if settings.FILTERS_NULL_CHOICE_VALUE in value:
+            value.append('')
+        return super().filter(qs, value)
```

Comment:
It feels like we need to further consider the implications of storing empty strings rather than null values for these fields. django-filter has native support for the later; replacing empty strings with `null` would obviate the need for a custom filter entirely AFAICT. Maybe that should be our preferred strategy.